content/logs/attributes_naming_convention.md

```diff
@@ -0,0 +1,139 @@
+## Why a naming convention for attributes?
+
+*In a nutshell:*
+- Various technologies tends to use different attribute names for the same meaning
+- This usually generates too many attributes which can lead to: user confusion and unability to correlate accross sources
+- Integrations rely on the following naming convention
+- We also advise you to use the following reference when you parse your custom formats & sources
+
+Centralizing logs from various technologies and applications tends to generate tens or hundreds of different attributes on the Log Management enviromnent. Especially when their are many users and usage working on it.
+
+This tends to generate confusion among users as - just to take a simple example - a client ip can be named `clientIP`, `client_ip_address`, `client.ip`, `http.client_ip` and many other combinations are possible...
+How do I then know which one corresponds to the http proxy logs I am currently analyzing? And how I am going to correlate this same proxy logs to the web application logs just behind it?
+
+Indeed, while people tend to name things slightly differently a URL, a client ip or a duration as the same meaning for everyone.
```

Comment:
Indeed, while people tend to name things slightly differently a URL, a client ip or a duration `have` the same meaning for everyone.